lighthouse-cli/test/fixtures/byte-efficiency/tester.html

```diff
@@ -55,6 +55,10 @@
   <!-- PASS(responsive): image is used at full size -->
   <img src="lighthouse-unoptimized.jpg">
 
+  <!-- FAIL(optimized): image is not optimized -->
+  <!-- PASS(responsive): image is used at full size -->
+  <img src="https://github.com/GoogleChrome/lighthouse/raw/751a84547bfff97d1f6c4533d1be2266908dfdc9/assets/lighthouse-logo_no-light.png">
```

Comment:
> is that ability worth writing and maintaining a second static server that you need to remember to bring up? :)

actually, now that I think about it, `static-server.js` already [does start up two servers](https://github.com/GoogleChrome/lighthouse/blob/cd82bc7d6e16881861cdfc4fea2f66c5a3d6ad01/lighthouse-cli/test/fixtures/static-server.js#L87-L88) :) The byte efficiency test is testing against `http://localhost:10200/byte-efficiency/tester.html`, so I believe you'll be able to use `http://localhost:10503` as a second origin.

It may still be a total pain, but it's worth a quick try to see if it will work